client/src/js/views/DashboardDogsListDogView.js

```diff
@@ -7,26 +8,38 @@ var DashboardDogsListDogView = Backbone.View.extend({
   template: _.template(DashboardDogsListDogTemplate),
 
   events: {
-    'change [name="checked_in"]': 'handleCheckedInChange',
-    'change [type="radio"]' : 'statusChange'
+    'click .button--check-in': 'handleCheckedInChange',
+    'change [type="radio"]' : 'statusChange',
+    'click .dropdown' : 'toggleDropdown'
   },
 
   initialize: function() {
     this.render();
-  },
 
+    this.listenTo(this.model, 'change:current_status', this.render);
+  },
+ 
   render: function() {
-    this.$el.html(this.template(this.model.attributes));
+    this.$el.html(this.template(this.model.toJSON()));
     return this;
   },
 
   handleCheckedInChange: function(e) {
-    this.model.set('checked_in', e.target.checked);
+    this.model.checkIn();
     this.render();
+    console.log('dog checked in');
   },
 
   statusChange: function(e) {
+    var val = $(e.target).val();
+    this.model.set('current_status', val);
+    this.toggleDropdown();
     console.log('radio button has been changed');
+  },
+
+  toggleDropdown: function(e) {
```

Comment:
Did the [Foundation Dropdown](http://foundation.zurb.com/sites/docs/dropdown-menu.html) not work here? I'd like to avoid custom JS here if possible since we already use Foundation as a boilerplate.